project.clj

```diff
@@ -4,45 +4,36 @@
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
   :dependencies [[org.clojure/clojure "1.8.0"]
-                 [org.clojure/clojurescript "1.9.89"]]
+                 [org.clojure/clojurescript "1.9.93"]]
```

Comment:
Does this need anything from cljs 1.9.93 that 1.9.89 doesn't have? Otherwise we try to stick to the latest release [published on clojurescript.org](https://github.com/clojure/clojurescript) which doesn't always match the latest one they've pushed to maven repo (unfortunately). That's still 1.9.89 as of this comment's posting.